bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -824,6 +1036,22 @@ private static void assertAdjudicationEquals(String expectedCategoryCode, BigDec
 	}
 
 	/**
+	 * @param expectedCategoryCode
+	 *            the expected {@link Coding#getCode()} of the
+	 *            {@link ItemAdjudicationComponent#getCategory()} to verify is
+	 *            not present
+	 * @param actuals
+	 *            the actual {@link ItemAdjudicationComponent}s to verify
+	 */
+	private static void assertAdjudicationNotPresent(String expectedCategoryCode,
+			List<ItemAdjudicationComponent> actuals) {
+		Optional<ItemAdjudicationComponent> adjudication = actuals.stream()
+				.filter(a -> DataTransformer.CODING_SYSTEM_ADJUDICATION_CMS.equals(a.getCategory().getSystem()))
+				.filter(a -> expectedCategoryCode.equals(a.getCategory().getCode())).findAny();
+		Assert.assertFalse(adjudication.isPresent());
+	}
```

Comment:
I think they are legible. Where the stream filtering gets difficult for me to parse is for instance in the transformInsertCarrierClaimEvent() test case below, where there is complex inline logic instead of using a helper method. 

```
BundleEntryComponent referrerEntry = claimBundle.getEntry().stream().filter(r -> {
			if (!(r.getResource() instanceof Practitioner))
				return false;
			Practitioner referrer = (Practitioner) r.getResource();
			return referrer.getIdentifier().stream()
					.filter(i -> DataTransformer.CODING_SYSTEM_NPI_US.equals(i.getSystem()))
					.filter(i -> record.referringPhysicianNpi.equals(i.getValue())).findAny().isPresent();
		}).findAny().get();
```